src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBrokerHelper.cs

```diff
@@ -246,7 +246,7 @@ public async Task InitiateBrokerHandshakeAsync(Activity callerActivity)
         private Handler GetPreferredLooper(Activity callerActivity)
         {
             var myLooper = Looper.MyLooper();
-            if (myLooper != null && callerActivity.MainLooper != myLooper)
+            if (myLooper != null && callerActivity != null && callerActivity.MainLooper != myLooper)
```

Comment:
This fix looks good to me.

Reply:
@jennyf19 - this piece of code is somewhat related to threading. The "looper" is smth equivalent to the UI / non-UI thread in WPF / WinForm apps. This piece of code tries to use the "looper" of the calling activity, or the main looper. The bug is that callerActivity seems that it can be null.

Reply:
@bgavrilMS yeah, i remember the looper.

Reply:
I had no idea such a thing exists. Funky name :)